training/templates/reg_success.html

```diff
@@ -94,7 +94,11 @@ <h5>Participant</h5>
       <br>
       <h5>FOSS</h5>
       <hr>
-      <p>{{participant_obj.event.foss}}</p>
+      <p>
+         {% for foss in participant_obj.event.course.foss.all %}
```

Comment:
```suggestion
         {% for foss in participant_obj.event.course.foss.all() %}
```
Are brackets required here?